packer/windows/scripts/start-agent.ps1

```diff
@@ -0,0 +1,93 @@
+function Set-InstanceHealth {
+  $Token = (Invoke-WebRequest -UseBasicParsing -Method Put -Headers @{'X-aws-ec2-metadata-token-ttl-seconds' = '60'} http://169.254.169.254/latest/api/token).content
```

Comment:
The `169.254.169.254` feels a bit magical. Is this the SSM enpoint?

Giving a name might help with understanding. Something like `const $SSMEnpoint="169.254.169.254"`

Reply:
No, that IP is the one you hit for instance metadata from inside an EC2 instance: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html